### PR TITLE
Updates to bootstrap docs and migration tool

### PR DESCRIPTION
**Pull request by orangejenny**
Updates based on today's team session. Review by commit.

### Safety story
Internal migration tool.

### Automated test coverage

In PR.

### QA Plan

no

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/hqwebapp/utils/bootstrap/changes_guide/css-form-group.md
Comment (jingcheng16) [inline corehq/apps/hqwebapp/utils/bootstrap/changes_guide/css-form-group.md:1]: Love this rewriting! >< Thank you!

File: corehq/apps/hqwebapp/utils/bootstrap/changes_guide/css-select-form-control.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Please replace form-control with form-select.
+
+In Bootstrap 3, selects used the `form-control` class.
+In Bootstrap5, selects use `form-select` instead.
+
+Please remove `form-control` from selects and add `form-select`.
```
Comment (jingcheng16) [inline corehq/apps/hqwebapp/utils/bootstrap/changes_guide/css-select-form-control.md:6]: Is this line convey the same meaning as first line?

**Review by jingcheng16 — APPROVED** [review]
Looks good to me. Just nit comment.

File: corehq/apps/hqwebapp/utils/bootstrap/changes_guide/css-select-form-control.md (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/hqwebapp/utils/bootstrap/changes_guide/css-select-form-control.md]: good point - 9966da6eeb7f28c54608766ac15df69ed91d1d2c

**Review by jingcheng16 — APPROVED** [review]
Thank you!

**Pull request merged**